lib/order.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/menu.rb'
+require './lib/verify.rb'
+
+class Order
+
+attr_reader :current_order, :current_menu, :item_no, :item_total, :collect_amount
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)